scripts/compareSize.js

```diff
@@ -0,0 +1,118 @@
+const fs = require('fs-extra');
+const path = require('path');
+const https = require('https');
+const Octokit = require('@octokit/rest');
+const octokit = new Octokit();
+const currentAppStatsFile = 'build-stats.txt';
+const currentPublishStatsFile = 'publish.json';
+const lastAppStatsFile = 'old-build-stats.txt';
+const lastPublishStatsFile = 'old-publish.json';
+
+compareBuildAppSize().catch(err => {
+  console.error(err.stack);
+  process.exit(1);
+});
+
+// Compares the example test app build size and the verdaccio published package sizes to the last release
+async function compareBuildAppSize() {
+  // Get the commit before the last publish commit
+  let res = await octokit.search.commits({q: 'Publish repo:adobe/react-spectrum'});
+  let {items} = res.data;
+  let commit = items && items[0]?.parents[0]?.sha;
+
+  if (commit) {
+    // Attempt to pull stats from last publish commit. If they don't exist, pull from a hardcoded commit (fallback until these records are generated for a first publish w/ this script)
+    let lastAppStatUrl = `https://reactspectrum.blob.core.windows.net/reactspectrum/${commit}/verdaccio/publish-stats/${currentAppStatsFile}`;
+    let lastPublishStatUrl = `https://reactspectrum.blob.core.windows.net/reactspectrum/${commit}/verdaccio/publish-stats/${currentPublishStatsFile}`;
+    let lastAppStatPath = path.join(__dirname, '..', lastAppStatsFile);
+    let lastPublishStatPath = path.join(__dirname, '..', lastPublishStatsFile);
+    await download(lastAppStatUrl, lastAppStatPath);
+    await download(lastPublishStatUrl, lastPublishStatPath);
+
+    if (!fs.existsSync(lastAppStatPath)) {
+      // TODO: placeholder until we get some real stats. Hardcoded URL pointing to a commit with actual data
```

Comment:
How are you planning on keeping this updated?

Reply:
I'll be deleting this fallback after we get this merged and a release is done. That way the script can always look up the commit before the last release and use that as the "lastApp/lastPublish" data point. The hardcoded fallback is only here so we can have an initial data point of comparison